testsuite/MDAnalysisTests/converters/test_rdkit.py

```diff
@@ -204,13 +189,12 @@ def test_monomer_info(self, pdb, sel_str, atom_index):
         sel = pdb.select_atoms(sel_str)
         umol = sel.convert_to.rdkit(NoImplicit=False)
         atom = umol.GetAtomWithIdx(atom_index)
-        mda_index = atom.GetIntProp("_MDAnalysis_index")
         mi = atom.GetMonomerInfo()
 
         for mda_attr, rd_attr in RDATTRIBUTES.items():
             rd_value = getattr(mi, "Get%s" % rd_attr)()
             if hasattr(sel, mda_attr):
-                mda_value = getattr(sel, mda_attr)[mda_index]
+                mda_value = getattr(sel, mda_attr)[atom_index]
```

Comment:
I find this somewhat hard to read, to be honest. I don't think this class is inherited, so could you please just type out the attributes you expect the AtomGroup to have?

Reply:
i.e.

```
assert sel.altLocs[atom_index] == monomer_info.GetAltLoc()
assert sel.names[atom_index] == monomer_info.GetName().strip()
```